TF_Settings_and_Tooling_Unity/Assets/TouchFree/ServiceShared/Scripts/Configuration/Managers/TFAppConfig.cs

```diff
@@ -55,7 +55,7 @@ public class ConfigurationState
 
         // CTI Settings
         public bool ctiEnabled = false;
-        public string ctiFilePath = Path.Combine(Application.streamingAssetsPath, "CallToInteract/AirPush_Portrait.mp4");
+        public string ctiFilePath = Path.Combine(ConfigFileUtils.ConfigFileDirectory, "CTI/AirPush_Portrait.mp4");
```

Comment:
Maybe change this to be Path.Combine for the "CTI/AirPush_Portrait.mp4" bit as well so that folder slashes are consistent

Reply:
Good spot